ls1intum/Artemis
# `Exam mode`: Fix setting an individual student exam working time

### Checklist
#### General


#### Client

### Motivation and Context
Bug noticed by @FlorianGlombik in https://github.com/ls1intum/Artemis/pull/4535#pullrequestreview-838805920.

### Description
The current form saves the working time in minutes, but it is actually in seconds. Therefore, the displayed duration was wrong.

### Steps for Testing
Prerequisites:
- 1 Exam

1. Go to a student exam.
2. Change the working time. It should be displayed correctly after the update as hours and minutes.
3. Optional: Also check the network traffic when refreshing the page for the JSON with the student exam (API-endpoint `/api/courses/$courseId/exams/$examId/student-exams/$studentExamId`). The `workingTime` field there contains the total number of seconds of working time. This should match the values displayed in the UI.

### Review Progress
#### Code Review
#### Manual Tests

### Test Coverage
unchanged

### Screenshots
![Screenshot 2022-01-03 at 21-02-22 Exam Management](https://user-images.githubusercontent.com/64250573/147980208-bcabd2bd-7d35-448f-991b-9fd057e6f3b9.png)
